frontend/src/app/organization/organization-create/organization-create.component.ts

```diff
@@ -0,0 +1,92 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+import { switchMap, tap } from 'rxjs/operators';
+import { Subscription } from 'rxjs/Subscription';
+
+import { AuthService } from '../../auth.service';
+import { User } from "../../user";
+import { Organization } from '../organization.model';
+import { OrganizationService } from '../organization.service';
+
+@Component({
+  selector: 'volontulo-organization-create',
+  templateUrl: './organization-create.component.html',
+  styleUrls: ['./organization-create.component.scss']
+})
+export class OrganizationCreateComponent implements OnInit, OnDestroy {
+  createForm: FormGroup;
+  id: number;
+  inEditMode = false;
+  message = '';
+  createSubscription: Subscription;
+  userSubscription: Subscription;
+  user: User;
+
+  constructor(
+    private authService: AuthService,
+    private activatedRoute: ActivatedRoute,
+    private fb: FormBuilder,
+    private organizationService: OrganizationService,
+    private router: Router,
+    ) {}
+
+  ngOnInit() {
+    this.userSubscription = this.authService.user$.subscribe(
```

Comment:
Since user$ is an observable of BehaviorSubject, it stores the last value and emits it immidiately on subscription. Because of that we can use take(1) rxJs operator.  Look at how it is used in organization.component.ts, line 57. Take(n) operator completes the observable after 'n' values have been read from it and unusbscribes automatically.